plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle.properties

```
@@ -75,6 +75,8 @@ dataTransfer.processor.sql.property.includeAutoGenerated.description=Include aut
 dataTransfer.processor.sql.property.escape.name=Escape character
 dataTransfer.processor.sql.property.escape.description=Character which escapes single quote character
 dataTransfer.processor.sql.property.extension.label = File extension
+dataTransfer.processor.sql.property.userTableName.name = Target table name
+dataTransfer.processor.sql.property.userTableName.description = You can specify the name of the target table in this field. It will be used to generate an INSERT statement.\nBy default, the name of the original table/tables is used.\nThe table name introduced into this field will be used as is.\nIf you want the table name also contains the scheme/catalog identifier, then you can prescribe it in this field by yourself.
```

Maybe we could write it shorter.
For example
```suggestion
dataTransfer.processor.sql.property.userTableName.description = You can specify the name of the target table. It will be used as is to generate an INSERT statement.\nIf you want the table name also contains the scheme or/and catalog identifier, you can prescribe it in this field by yourself.\nBy default, the name of the original table is used.
```